deploy/olm/watches.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- version: v1alpha1
+  group: operator.cert-manager.io
+  kind: CertManager
+  chart: helm-charts/cert-manager
+  overrideValues:
+    image.repository: registry.connect.redhat.com/jetstack/cert-manager-controller
+    image.tag: v0.15.0-alpha1.-ubi
```

Comment:
`alpha.1`

Reply:
Also the other Dockerfiles etc in this PR reference `RUN wget https://charts.jetstack.io/charts/cert-manager-v0.15.0-alpha.0.tgz` (i.e. alpha.0) - it seems odd that these are different